src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/commands.py

```diff
@@ -0,0 +1,119 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._util import (get_monitor_alert_rules_operation, get_monitor_alert_rule_incidents_operation,
+                    get_monitor_log_profiles_operation, get_monitor_autoscale_settings_operation,
+                    get_monitor_service_diagnostic_settings_operation,
+                    get_monitor_usage_metrics_operation, get_monitor_event_categories_operation,
+                    get_monitor_events_operation, get_monitor_tenant_events_operation,
+                    get_monitor_metric_definitions_operation, get_monitor_metrics_operation,
+                    ServiceGroup, create_service_adapter)
+
+# MANAGEMENT COMMANDS
+alert_rules_operations = create_service_adapter(
+    'azure.cli.command_modules.monitor.sdk.operations.alert_rules_operations',
+    'AlertRulesOperations')
```

Comment:
This is another way of just doing `modpath.format(a, b, c)` like in the VM module (https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py#L19).

I don't really see this implementation as *simpler* than what's in the VM module as I now have to follow `create_service_adapter` to see what that does.